tests/unit_tests/libs_tests/ECS_tests/Entity_tests/CMakeLists.txt

```diff
@@ -1,19 +1,11 @@
-
-## Setting Variables ##
-set (INCLUDE_ROOT ${PROJECT_SOURCE_DIR}/tests/unit_tests/libs_tests/ECS_tests/Entity_tests)
 set (SRC_ROOT ${PROJECT_SOURCE_DIR}/tests/unit_tests/libs_tests/ECS_tests/Entity_tests)
 
 set(TEST_SRC_ROOT
     ${SRC_ROOT}/Entity_class_tests.cpp
 )
 
-## Documented Example ##
```

Comment:
Do NOT Remove Cmake documentation

Reply:
IT's the template documentation. She has nothing to do here